### PR TITLE
Added installed debian packages checker

### PR DESCRIPTION
**Pull request by constmonica**
Added a new functionality to get_apt_info
The 'apt-package' report file generated gathers information about Debian packages, their version and architecture.
I have tested the diagnostic report tool on Raspberry Pi 4.

## PR Type
- [ ] Bug fix (change that fixes an issue)
- [x] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [x] I have performed a self-review of the changes
- [ ] I have commented my code, at least hard-to-understand parts
- [x] I have tested the modification and rebuild the diagnostic report tool
- [x] I have signed off all commits from this PR
- [ ] I have updated the documentation (wiki pages, ReadMe etc)

### CODE REVIEW

**Comment by SRaus** [comment]
There are some other issues open. Can you check what else can be fixed:
1. It seems that the resulted archive does not contain what it should (both kuiper versions):
image

2. The new introduced 'Installed Debian package checker' adds an warnings in archive (in kuiper2.0)
image
And an error in current kuiper:
image
Maybe there can be added a condition to check the file / package repo existence first.

3. The application is not added in "Application Menu" by 'make install' in Kuiper2.0. For the current kuiper, it is added in Menu, but trying to open it from there - as sudo: it does not open, as other user - it opens but you cannot do anything, since it requires sudo rights.
image

4. In both Kuiper versions there is a Critical Warning about GTK version 
image

File: adi_diagnostic_report [hunk 1]
```diff
@@ -291,12 +291,15 @@ def get_sys_bus():
...
-    matches = re.findall(pattern, result)
+    if os.path.isfile(file_path):
+        grep_command = f"grep '^Package: ' {file_path} | awk '{{print $2}}'"
+        packages = subprocess.check_output(grep_command, shell=True).decode('utf-8').splitlines()
+        dpkg_command = f"dpkg -s {' '.join(packages)} 2>/dev/null | grep -B 1 -A 4 'ok installed'"
+        result = subprocess.check_output(dpkg_command, shell=True).decode('utf-8')
+        pattern = r"(Package|Architecture|Version):\s*(.+)"
+        matches = re.findall(pattern, result)
+    else:
+        return {'apt-pack': "The file path doesn't exist."}
```
Comment (SRaus) [inline adi_diagnostic_report]: Can you also print the value of `file_path` here?

File: adi-diagnostic-report.desktop.in [hunk 2]
```diff
@@ -3,6 +3,6 @@ Exec=@PREFIX@/bin/adi_diagnostic_report --gui
 Icon=utilities-system-monitor
 Terminal=false
 Type=Application
-Categories=System
+Categories=Science
```
Comment (SRaus) [inline adi-diagnostic-report.desktop.in:6]: Isn't there a dedicated category like "Analog Devices tools" ?

**Review by SRaus — CHANGES_REQUESTED** [review]

File: adi_diagnostic_report (hunk 1 above)
Comment (constmonica) [reply adi_diagnostic_report]: done

File: adi-diagnostic-report.desktop.in (hunk 2 above)
Comment (constmonica) [reply adi-diagnostic-report.desktop.in:6]: 'Science' is the standard name, changed in kuiper to Analog Devices Tools

**Review by SRaus — APPROVED** [review]

**Pull request merged**